eslint.config.ts

```diff
@@ -145,7 +145,11 @@ const config = tsEslint.config([
     languageOptions: {
       parserOptions: {
         projectService: {
-          allowDefaultProject: ['eslint.config.ts', 'knip.ts'],
+          allowDefaultProject: [
+            'eslint.config.ts',
+            'knip.ts',
+            'node_package/tests/serverRenderReactComponent.test.ts',
```

Comment:
```suggestion
            'node_package/tests/**/*.test.ts',
```
we don't want to edit it every time a new TS test is added :) Maybe even `{ts,tsx}`.

Reply:
Run `nps format` after this in case it fits on one line.

Reply:
I've got the error
```
error  Parsing error: allowDefaultProject glob 'node_package/tests/**/*.test.ts' contains a disallowed '**'.
```
So, I added `node_package/tests/*.test.ts` instead.